spec/requests/users/sessions_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe Users::SessionsController, type: :request do
+  describe 'Sign-in redirect_to' do
+    let(:user) { create(:user) }
+    let(:curriculum_id) { create(:curriculum).id }
+
+    context 'when redirect_to is not present as parameter' do
+      def perform
+        sign_in user
+        post user_session_url
+      end
+
+      it 'redirects to the root_path' do
+        expect(perform).to redirect_to(root_path)
```

Comment:
I tried with this block, but the `redirect_to` matcher doesn't support blocks as input. What can be done to avoid the perform method is to include the lines directly in the `it`:
```
it 'redirects to the root_path' do
  sign_in user
  post user_session_url  
  expect(response).to redirect_to(root_path)
end
```